noir-projects/aztec-nr/aztec/src/context/call_interfaces.nr

```diff
@@ -753,11 +769,234 @@ impl<let N: u32> UtilityVoidCallInterface<N> {
         self.selector
     }
 
-    pub fn get_name(self) -> str<N> {
+    pub fn get_name(self) -> str<M> {
         self.name
     }
 
     pub fn get_contract_address(self) -> AztecAddress {
         self.target_contract
     }
 }
+
+pub trait TXEPublicCallInterface<T, let N: u32> {
```

Comment:
Slightly worried this pollutes the interfaces and anyone could call them from regular contracts (even if the oracles would fail). What happens if we add these to the test env itself? Wouldn't the user have to explicitly import the test fails for these to become available?